crates/forge/bin/cmd/create.rs

```diff
@@ -823,40 +841,17 @@ where
     P: Provider<T, AnyNetwork>,
     T: Transport + Clone,
 {
-    /// Set zksync's factory deps.
-    pub fn set_zk_factory_deps(mut self, deps: Vec<Vec<u8>>) -> Self {
-        self.zk_factory_deps = Some(deps);
-        self
-    }
-
-    /// Set zksync's paymaster params.
-    pub fn set_zk_paymaster_params(mut self, params: Option<PaymasterParams>) -> Self {
```

Comment:
Can we port this over to the`ZkDeployer`? It's a shame that we need to unpack the option to set the value only for the paymaster

Reply:
Where are we doing this exactly?

Reply:
Not sure if I see the value in that, we can't use `unwrap_or_default()` or `with_paymaster(..)` as we require the value to be `None` if paymaster params aren't present, which is again idiomatic rust via `if let Some(..)`. If we go with helper methods in `ZkDeployer`, we'd just end up creating wrapper methods for `Option`'s that around the already defined methods on `alloy_zksync`.